engine/storage/volume/src/main/java/org/apache/cloudstack/storage/datastore/lifecycle/BasePrimaryDataStoreLifeCycleImpl.java

```diff
@@ -39,9 +38,12 @@
 import com.cloud.storage.StoragePoolHostVO;
 import com.cloud.storage.dao.StoragePoolHostDao;
 import com.cloud.utils.Pair;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class BasePrimaryDataStoreLifeCycleImpl {
-    private static final Logger s_logger = Logger.getLogger(BasePrimaryDataStoreLifeCycleImpl.class);
+    private static final Logger logger = LogManager.getLogger(BasePrimaryDataStoreLifeCycleImpl.class);
```

Comment:
```suggestion
    protected Logger logger = LogManager.getLogger(BasePrimaryDataStoreLifeCycleImpl.class);
```
Then we can remove the logger definitions on the inheritor classes

Reply:
Isn't it better to have different loggers for inherited classes to identify the source class of a log line?

Reply:
can use `LogManager.getLogger(getClass())` instead 
@vishesh92

Reply:
Sorry, there was a typo in my suggestion. I fixed it. Now the inheritors will log their own classes. You can check this comment regarding the OOP inheritance concept as well: https://github.com/apache/cloudstack/pull/7131#issuecomment-1491900877